Documentation/applications/graphics/lvgl/index.rst

```diff
@@ -0,0 +1,27 @@
+====
+LVGL
+====
+
+Usage
+-----
+
+Import with ``#include <lvgl/lvgl.h>`` or ``#include <lvgl.h>``.
```

Comment:
I think the original documentation is incorrect, the right should be only <lvgl/lvgl.h>

Reply:
both should work?